Controllers/FibonacciController.cs

```diff
@@ -27,20 +27,23 @@ public async Task<int> CalculateFibonacciAsync(int index = 0)
         // currentSpan.SetAttribute("parameter.index", index);
 
         if (index == 0 | index == 1)
+        {
             return 0;
+        }
+        
         if (index == 2)
+        {
             return 1;
+        }
 
         var resOne = await GetNext(index - 1);
         var resTwo = await GetNext(index - 2);
 
         // CUSTOM SPAN (3 sections of code to uncomment, 4 lines total)
```

Comment:
update comment too

Reply:
Oh, and needs a semicolon. Too much go programming recently 🙂